Sources/ENCore/App/Features/DeveloperMenu/DeveloperMenuViewController.swift

```diff
@@ -369,44 +365,40 @@ final class DeveloperMenuViewController: ViewController, DeveloperMenuViewContro
         present(activityViewController, animated: true, completion: nil)
     }
 
-    private var fetchAndProcessCancellable: AnyCancellable?
+    private var fetchAndProcessDisposable: Disposable?
 
     private func fetchAndProcessKeySets() {
-        guard fetchAndProcessCancellable == nil else {
-            fetchAndProcessCancellable?.cancel()
-            fetchAndProcessCancellable = nil
-
+        guard fetchAndProcessDisposable == nil else {
+            fetchAndProcessDisposable?.dispose()
+            fetchAndProcessDisposable = nil
             return
         }
 
         internalView.tableView.reloadData()
 
-        fetchAndProcessCancellable = exposureController
-            .fetchAndProcessExposureKeySets()
-            .sink(
-                receiveCompletion: { [weak self] _ in
-                    self?.fetchAndProcessCancellable = nil
+        fetchAndProcessDisposable = exposureController
+            .fetchAndProcessExposureKeySets().subscribe { _ in
```

Comment:
Is the change to capturing `self` strongly here, and in the closures below, intentional?